pattern-library/html/boilerplate-header+footer+content+nav.html

```diff
@@ -0,0 +1,113 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer
+
+NOTES:
+
+* base template with the additon of how to handle an app-centric header/footer semantically.
+* base template with the additon of how to handle different types of navigation (user vs. app-centric).
+* base template with addition of how to start organizing main vs. supplementary/aside content
+* this assumes that the main and aside content are next to each other in the document's flow. This does not neccessarily have to be the case, but the same markup could be used in more complex cases.
+-->
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8">
+        <!-- always force latest IE rendering engine or request chrome rrame -->
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
```

Comment:
If you're aiming for strict validation here, it's best to define these in the headers for the document, not in the page source itself.  IE=edge,chrome=1 is not a valid value for the content attribute here.